compiler/rustc_resolve/src/late/diagnostics.rs

```diff
@@ -3114,9 +3114,9 @@ impl<'ast, 'ra: 'ast, 'tcx> LateResolutionVisitor<'_, 'ast, 'ra, 'tcx> {
         }
 
         let existing_name = match &in_scope_lifetimes[..] {
-            [] => Symbol::intern("'a"),
+            [] => sym::apostrophe_a,
             [(existing, _)] => existing.name,
-            _ => Symbol::intern("'lifetime"),
+            _ => sym::apostrophe_lifetime,
```

Comment:
Suggestion: please revert this, only in diagnostics path, less readable than before.